environment.yml

```diff
@@ -22,33 +22,41 @@ dependencies:
   - jupyter
   # - jupyter_core
   # - jupyter_console
-  - keras
+  # - keras
   # - libgcc
   - lxml
-  - matplotlib
-  - mysqlclient
-  - nbconvert
+  # - matplotlib
+  # - mysqlclient
+  # - nbconvert
   - nltk
   - pandas
-  - pandoc
+  # - pandoc
   - pip
-  - plotly
+  # - plotly
   - pre-commit
-  - prompt_toolkit<3.0
+  # - prompt_toolkit<3.0
   - psycopg2
   - pyscaffold
   - pytest-cov
-  - python-levenshtein
+
   - python=3.7
-  - redis-py
+  - pytorch
+  # conda install pytorch cudatoolkit=10.1 -c pytorch
+
+  # - redis-py
   - regex  # ==2018.1.10
+  - seaborn
   - scipy
   - spacy>=2.2.2
-  - sqlparse
-  - tensorflow==2.0  # ==1.13.1
+  - tqdm
+  - watchdog
+  # - sqlparse
+  # - tensorflow==2.0  # ==1.13.1
 
   - pip:
-    - wikipedia-api
+    # - apex
     - environment
     - fuzzywuzzy
     # - nlpia
+    - simpletransformers
```

Comment:
We shouldn't need the simpletransformers library anymore. I removed the references to it in qa_bots.